parser.go

```diff
@@ -52,6 +52,10 @@ var (
 	pNot           = newTokenPattern("!", "\\!")
 	pEqual         = newTokenPattern("==", "\\=\\=")
 	pNotEqual      = newTokenPattern("!=", "\\!\\=")
+	pGT            = newTokenPattern(">", "\\>")
```

Comment:
`pGt` for consistency

Reply:
I believe the official guidance is to make initialisms either all lowercase, all uppercase--but never mixed:

> Words in names that are initialisms or acronyms (e.g. "URL" or "NATO") have a consistent case. For example, "URL" should appear as "URL" or "url" (as in "urlPony", or "URLPony"), never as "Url". As an example: ServeHTTP not ServeHttp. For identifiers with multiple initialized "words", use for example "xmlHTTPRequest" or "XMLHTTPRequest".

https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
I think a good solution here is to change to `pGreaterThan`, etc., which is more literate anyway. :-)